build-scripts/ci_parser/ci_parser.py

```diff
@@ -68,50 +85,50 @@ def main():
 
 def extract_junit_from_test_run(input_dir: str) -> Dict[str, JUnitTestSuite]:
     """
-    For a given input input_dir, will find all .gz files in that tree, extract files from them preserving input_dir structure
+    For a given input input_dir, will find all .xml files in that tree, extract files from them preserving input_dir structure
     and parse out all found junit test results into the global test result containers.
-    :param input_dir: Input directory to recursively search for .gz files
+    :param input_dir: Input directory to recursively search for .xml files
     """
 
     # Skip archives that we know exist but don't have results we want.
-    gz_exclusions = ['split', 'result_details']
+    xml_exclusions = ['split', 'result_details']
 
     # Inclusions win over exclusions right now but are empty by default.
 
     # TODO: Make this a command-line regex? Used for debugging; could use to parse just a certain subset of suites.
-    gz_inclusions = None  # type: Optional[List[str]]
-    # gz_inclusions = ['python']
+    xml_inclusions = None  # type: Optional[List[str]]
+    # xml_inclusions = ['python']
 
     # TODO: Make this a command-line debug flag? Used for debugging
     debug_file = None  # type: Optional[str]
-    # debug_file = 'jvm17-utests_archive.tar.gz'
+    # debug_file = 'jvm17-utests_archive.tar.xml'
     if debug_file is not None:
-        gz_files = [str(file) for file in Path(input_dir).rglob('*.gz') if debug_file in str(file)]
-    elif gz_inclusions is not None:
-        gz_files = [str(file) for file in Path(input_dir).rglob('*.gz') if any(x in str(file) for x in gz_inclusions)]
+        xml_files = [str(file) for file in Path(input_dir).rglob('*.xml') if debug_file in str(file)]
+    elif xml_inclusions is not None:
+        xml_files = [str(file) for file in Path(input_dir).rglob('*.xml') if any(x in str(file) for x in xml_inclusions)]
     else:
-        gz_files = [str(file) for file in Path(input_dir).rglob('*.gz') if not any(x in str(file) for x in gz_exclusions)]
-    check_file_condition(lambda: len(gz_files) != 0, f'Found 0 .gz files in path: {input_dir}. Cannot proceed with .xml extraction.')
-    logger.debug(f'Extracting .xml from {len(gz_files)} gzip files from path: {input_dir}')
+        xml_files = [str(file) for file in Path(input_dir).rglob('*.xml') if not any(x in str(file) for x in xml_exclusions)]
```

Comment:
In the non-gz case since you're dealing with single files, I'd remove the debug, inclusion, and exclusion functionality entirely. Those are useful when you have hundreds of tests in a single logical entity (i.e. a .gz file) you can bypass, but in the case of single files it's probably too laborious.